zpages/internal/templates/rpcz.html

```diff
@@ -28,24 +28,24 @@
     <tr>
         <td><b>{{.Method}}</b></td>
         <td></td>
-        <td align="right">{{.CountMinute|count}}</td>
```

Comment:
Where are we getting these column changes from?

Reply:
Oh I see, from the latest spec changes as per https://github.com/census-instrumentation/opencensus-go/commit/85464adb436a3dcfbca051c298ddd2cbc97c5e68